book/src/themes.md

```diff
@@ -151,7 +151,9 @@ We use a similar set of scopes as
   - `operator` - `or`, `in`
   - `directive` - Preprocessor directives (`#if` in C) 
   - `function` - `fn`, `func`
-  - `storage` - Keywords that affect the storage of a variable, function or data structure `static`, `mut`, `const`, `ref`
+  - `storage` - Keywords that affect the storage of variables, functions, data structures etc. 
+    - `type` - Keywords for declaring the storage of variables, functions, data structures  etc. eg `let`, `trait`, `struct` in rust 
```

Comment:
Let's not duplicate the description here - the fallback behavior implies that `keyword.store.type` is a subset of `keyword.storage`. Textmate's description is good:

> the type of something, `class`, `function`, `int`, `var`, etc.

Examples are useful but we shouldn't use an example that isn't actually captured with the scope like the Rust ones listed here. Ones that are common between languages like the ones textmate lists are good, or the new ones from this change